.github/workflows/release-sdk.yaml

```diff
@@ -0,0 +1,31 @@
+name: Release Python SDK
+
+on:
+  release:
+    types: [published]
+
+env:
+  TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+  TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Release Python SDK
```

Comment:
I think the name should be `Setup Python`?

Reply:
Good catch. Fixed